ambry-store/src/main/java/com/github/ambry/store/BlobStoreStats.java

```diff
@@ -128,17 +132,19 @@
     this.metrics = metrics;
     this.enableBucketForLogSegmentReports = enableBucketForLogSegmentReports;
     this.enablePurgeDeleteTombstone = enablePurgeDeleteTombstone;
+    this.longLiveTaskScheduler = longLiveTaskScheduler;
+    this.shortLiveTaskScheduler = shortLiveTaskScheduler;
 
     if (bucketCount > 0) {
       indexScanner = new IndexScanner();
-      longLiveTaskScheduler.scheduleAtFixedRate(indexScanner, 0,
+      indexScannerScheduledFuture = longLiveTaskScheduler.scheduleAtFixedRate(indexScanner, 0,
           TimeUnit.MILLISECONDS.toSeconds(bucketCount * bucketSpanTimeInMs), TimeUnit.SECONDS);
```

Comment:
why convert from milliseconds back to seconds here as opposed to setting the unit arg to milliseconds?